.github/workflows/clear-artifacts.yml

```diff
@@ -0,0 +1,77 @@
+name: Clear artifacts
+
+on:
+  workflow_dispatch:
+    inputs:
+      from_date:
+        description: "Start date (YYYY-MM-DD)"
+        required: false
+      to_date:
+        description: "End date (YYYY-MM-DD)"
+        required: false
+      older_than_date:
```

Comment:
older_than_date nu e același lucru cu a seta doar to_date? zic în ideea să simplificăm: poți seta mai simplu `FROM` și `TO`:
```bash
FROM="${{ github.event.inputs.from_date || "1970-01-01" }}"
TO="${{ github.event.inputs.to_date || $(date +%Y-%m-%d) }}" # aici nu sunt sigur dacă e ok sintaxa
```
și așa se elimină multe din if/else-urile actuale